docs/queries.md

```diff
@@ -97,11 +97,7 @@ await Person.
 
 A full list of [Query helper functions can be found in the API docs](api/query.html).
 
-<h2 id="queries-are-not-promises">
-  <a href="#queries-are-not-promises">
-    Queries are Not Promises
-  </a>
-</h2>
+## Queries are Not Promises
```

Comment:
I confirmed that the custom id isn't necessary here

Reply:
other places also define still define even if not necessary currently, maybe this should then also be like the others for now?